modules/release-notes/snip-feature-rhidp-2888.adoc

```diff
@@ -0,0 +1,7 @@
+[id="feature-rhidp-2888"]
+= Visualize Virtual Machine nodes on the Topology plugin
+
+With this update, you can visualize the Virtual Machine nodes deployed on the cluster through the Topology plugin.
+
```

Comment:
Should we link to the product docs from here...it will be _Ch 6.2.4. Viewing virtual machines_ in the _Configuring dynamic plugins_ guide.

Reply:
I can't see Configuring dynamic plugins guide in the list in https://content-stage.docs.redhat.com/en/documentation/red_hat_developer_hub/1.3/.
Is in missing in Pantheon?

Reply:
This guide is not merged yet. Add the link once after merge.